2023/AzureMeetupTorino/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
```

Comment:
I need the GH user identities for the person or people who are allowed to edit this community folder, please! @alberto-annunziata90

Reply:
I added your user as the owner of this folder. You can add more if you like, because you are the owner ;). Then just send us a new PR!